### ISSUE TITLE
Wrong memory usage on Odroid U3 running Debian Buster

### ISSUE THREAD

**Issue by zoof** [aristocratos/bashtop#80]
**Describe the bug**

I am running Debian Buster on an Odroid U3 and memory usage does not show up correctly (100% despite substantial "free" memory displayed).  (See below.)  It is admittedly running an old kernel (3.8.13.23) because Hardkernel no longer supports the device and I have not bothered to install a community developed kernel update.

![Screenshot_2020-04-30_10-02-45](https://user-images.githubusercontent.com/1737483/80719645-c7996580-8ac9-11ea-90b6-2966e751586c.png)

**To Reproduce**

Run `bashtop`.

**Info (please complete the following information):**
 - Bashtop version: 0.8.17
 - Linux distribution and version: Debian Buster
 - Terminal used: lxterminal
 - Font used: 
 - Bash version, "bash --version" (version 4.4 or above is required): 5.0.3(1)-release (arm-unknown-linux-gnueabihf)
 - Locales: output of "locale -v"

```
$ locale -v
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=en_US.UTF-8
```
**Additional context**

contents of `$HOME/.config/bashtop/error.log`
 
```
New instance of bashtop version: 0.8.17 Pid: 9303
02:21:36 PM ERROR: On line 1497 
02:21:45 PM ERROR: On line 1497 (MUTED!)
```

**Comment by aristocratos** [comment]
Would you mind posting the contents of `/proc/meminfo` since all memory data is collected from there.

**Comment by aristocratos** [comment]
Also, use the bug report template when reporting bugs.

**Comment by zoof** [comment]
```
$ cat /proc/meminfo 
MemTotal:        2071408 kB
MemFree:         1513940 kB
Buffers:           23072 kB
Cached:           322752 kB
SwapCached:            0 kB
Active:           314236 kB
Inactive:         190096 kB
Active(anon):     159736 kB
Inactive(anon):     8932 kB
Active(file):     154500 kB
Inactive(file):   181164 kB
Unevictable:           0 kB
Mlocked:               0 kB
HighTotal:       1317888 kB
HighFree:         835376 kB
LowTotal:         753520 kB
LowFree:          678564 kB
SwapTotal:             0 kB
SwapFree:              0 kB
Dirty:                16 kB
Writeback:             0 kB
AnonPages:        158592 kB
Mapped:            29488 kB
Shmem:             10164 kB
Slab:              40932 kB
SReclaimable:      30016 kB
SUnreclaim:        10916 kB
KernelStack:        1032 kB
PageTables:         2020 kB
NFS_Unstable:          0 kB
Bounce:                0 kB
WritebackTmp:          0 kB
CommitLimit:     1035704 kB
Committed_AS:    1068016 kB
VmallocTotal:     245760 kB
VmallocUsed:        2904 kB
VmallocChunk:     105276 kB
```

**Comment by zoof** [comment]
Updated initial comment and have also tried the current version of `bashtop`.

**Comment by aristocratos** [comment]
You're missing values for "MemAvailable" which is messing up the calculations.
It was added in kernel version 3.14.
But I would recommend updating the kernel since this seems to be a pretty rare case of a very outdated kernel.

**Comment by aristocratos** [comment]
Can add a error check for when I have time.

**Comment by aristocratos** [comment]
@zoof  Fixed in v0.8.27
